roboflow/core/version.py

```diff
@@ -312,16 +312,18 @@ def deploy(self, model_type: str, model_path: str) -> None:
             model_path (str): File path to model weights to be uploaded
         """
 
-        supported_models = ["yolov8", "yolov5", "yolov7-seg"]
+        supported_models = ["yolov5", "yolov7", "yolov8"]
```

Comment:
Is this up to date with what we ended up deciding to support?

Reply:
On convert server side that is?

Reply:
I updated the list and changed yolov7 to yolov7-seg.  This will make sure yolov7 and yolov7-cls are rejected.  This logic may need some rework though since yolov5-cls would not be rejected but is also not supported.